### PR TITLE
feat: add mutations and extend filter inputs for `INNM`, `INNMDosage` and `Medication` types

### PR DESCRIPTION
**Pull request by svetavedmed**
edenlabllc/ehealth.api#4187
edenlabllc/ehealth.api#4185
edenlabllc/ehealth.api#4183

### CODE REVIEW

File: __schema__/medications.graphql [hunk 1]
```diff
@@ -60,6 +70,108 @@ type MedicationEdge {
...
+  "A minimum quantity that could be sold"
+  packageMinQty: Int
+  "Quantity of pills in medication"
+  packageQty: Int
+}
+
+"""
+Input for `container`.
+"""
+input containerInput {
```
Comment (kiurchv) [inline __schema__/medications.graphql]: Input names should be in upper camel case

```suggestion
input ContainerInput {
```

File: __schema__/medications.graphql [hunk 2]
```diff
@@ -60,6 +70,108 @@ type MedicationEdge {
...
+  "Medication name"
+  name: String!
+  "A minimum quantity that could be sold"
+  packageMinQty: Int
+  "Quantity of pills in medication"
+  packageQty: Int
+}
+
+"""
+Input for `container`.
```
Comment (kiurchv) [inline __schema__/medications.graphql]:
```suggestion
Input for `Container`.
```

File: __schema__/medications.graphql [hunk 3]
```diff
@@ -60,6 +70,108 @@ type MedicationEdge {
...
+  "Denumerator unit"
+  denumeratorUnit: MedicationUnit!
+  "Denumerator value"
+  denumeratorValue: String!
+}
+
+"""
+Input for `medicationIngredient` mutation.
+"""
+input medicationIngredientInput {
```
Comment (kiurchv) [inline __schema__/medications.graphql]:
```suggestion
input CreateMedicationIngredientInput {
```

File: __schema__/medications.graphql [hunk 4]
```diff
@@ -60,6 +70,108 @@ type MedicationEdge {
...
+  "Is the component primary or not"
+  isPrimary: Boolean!
+  "International Nonproprietary Name"
+  innmDosage: CreateINNMDosageInput!
+}
+
+"""
+Input for `Dosage` mutation.
+"""
+input DosageInput {
```
Comment (kiurchv) [inline __schema__/medications.graphql]:
```suggestion
input CreateDosageInput {
```

File: __schema__/medications.graphql [hunk 5]
```diff
@@ -60,6 +70,108 @@ type MedicationEdge {
...
+  "Denumerator unit"
+  denumeratorUnit: MedicationUnit!
+  "Denumerator value"
+  denumeratorValue: String!
+}
+
+"""
+Input for `manufacturer` mutation
+"""
+input ManufacturerInput {
```
Comment (kiurchv) [inline __schema__/medications.graphql]:
```suggestion
input CreateManufacturerInput {
```

File: __schema__/medications.graphql [hunk 6]
```diff
@@ -96,6 +208,8 @@ type Medication implements Node {
   insertedAt: DateTime!
   "Date and time when record was updated"
   updatedAt: DateTime!
+  "Daily dose of medication, recommended by WHO"
```
Comment (kiurchv) [inline __schema__/medications.graphql]: Let's move this field above timestamp fields

File: __schema__/innmDosages.graphql [hunk 7]
```diff
@@ -42,6 +56,54 @@ type INNMDosageEdge {
...
+  "MedicationForm form"
+  form: MedicationForm!
+  "INNMDosage ingredients"
+  ingredients: [INNMDosageIngredientInput]!
+}
+
+"""
+Input for `INNMDosageIngredient` mutation.
+"""
+input INNMDosageIngredientInput {
```
Comment (kiurchv) [inline __schema__/innmDosages.graphql]: I propose to prefix all related to create/update input types. @edenlabllc/dev-frontend, @PavelVesnin, @progsmile   your thoughts?

```suggestion
input CreateINNMDosageIngredientInput {
```

Comment (progsmile) [reply __schema__/innmDosages.graphql]: Totally agree!

File: __schema__/innmDosages.graphql [hunk 8]
```diff
@@ -8,16 +8,43 @@ input INNMDosageFilter {
...
+  ingredients: INNMDosageIngredientFilter
+  "is_active status."
+  isActive: Boolean
+}
+
+"""
+INNMDosageIngredient filter.
+"""
+input INNMDosageIngredientFilter {
+  # TODO: When changed names of inputs they became unlogical here. We need to decide what filters to use?
```
Comment (progsmile) [inline __schema__/innmDosages.graphql]: @svetavedmed @kiurchv

**Pull request merged**